components/server/ee/src/prebuilds/prebuild-manager.ts

```
@@ -128,6 +128,11 @@ export class PrebuildManager {
             if (user.blocked) {
                 throw new Error(`Blocked users cannot start prebuilds (${user.name})`);
             }
+            if (!project) {
+                throw new Error(
```

Can we throw a User Error (Invalid Argument) here?

---

Good idea! I think this can be done. Let me try.